src/axom/quest/readers/C2CReader.hpp

```diff
@@ -67,9 +67,25 @@ class C2CReader
    * 
    * Knot spans are the sub-intervals within a spline
    */
-  void getLinearMesh(mint::UnstructuredMesh<mint::SINGLE_SHAPE>* mesh,
+  void getLinearMesh(mint::UnstructuredMesh<mint::SINGLE_SHAPE> *mesh,
```

Comment:
Should this function also get a transformation matrix?
Also, for consistency, should it optionally compute the revolved volume?